cmd/minikube/main.go

```diff
@@ -31,12 +32,17 @@ const minikubeEnableProfile = "MINIKUBE_ENABLE_PROFILING"
 
 func main() {
 	defer glog.Flush()
-
 	if os.Getenv(minikubeEnableProfile) == "1" {
 		defer profile.Start(profile.TraceProfile).Stop()
 	}
 	if os.Getenv(constants.IsMinikubeChildProcess) == "" {
 		machine.StartDriver()
 	}
+	console.SetOutFile(os.Stdout)
+	console.SetErrFile(os.Stderr)
+	err := console.SetPreferredLanguage(os.Getenv("LANG"))
+	if err != nil {
+		glog.Warningf("unable to detect language: %v", err)
```

Comment:
can the user do anything about this? what does it mean? are we falling back to default English?

Reply:
Small nitpick here is that LANG is the locale (while LANGUAGE is the language)

    LANG=sv_SE.UTF-8
    LANGUAGE=sv

Reply:
LANG settings are incredibly complicated: LANG, LANGUAGE, and LC_* all control specific locale settings. LANG is the fallback.

To reduce complexity, I've removed the call to SetPreferredLanguage here. I'll re-introduce it once we have a way of loading alternative languages.